### PR TITLE
Fixes to build process

### PR DESCRIPTION
**Pull request by AlirezaHaghshenas**
# 🌮 Taqueria PR

## 🪁 Description

_Explain it to me like I'm 5! Keep this short and descriptive_

## 🪂 Pre-Merge Checklist (Definition of Done)

🚦 Required to merge:

- [ ] ⛱️ I have completed this PR template in full and updated the title appropriately
- [ ] ⛵ My code builds cleanly, and I have manually tested the changes
- [ ] 🏄‍♂️ Another team member has built this branch and done manual testing on the change
- [ ] 🏖️ New and existing unit tests pass locally and in CI
- [ ] 🔱 The test plan has been implemented and verified by an SDET
- [ ] 🦀 Automated tests have been written and added to this PR
- [ ] 🐬 I have commented my code, particularly in hard-to-understand areas
- [ ] 🤿 Corresponding changes have been made to all documentation
- [ ] 🐚 Required changes to the VScE have been made
- [ ] 🪸 Required updates to scaffolds have been made
- [ ] 🚢 The release checklist has been completed

## 🛩️ Summary of Changes

_Please include a summary of the changes to the codebase._ _Please also include relevant motivation and context for the change_
_(e.g. what was the existing behaviour, why did it break, etc.)_

## 🎢 Test Plan

_Please describe the testing strategy and plan for this PR. Keep this lightweight and anticipate any testing challenges_

## 🛸 Type of Change

- [ ] 🧽 Chore ➾
- [ ] 🛠️ Fix ➾
- [ ] ✨ Feature ➾
- [ ] 👷 Refactor ➾
- [ ] 🧪 Pre-Release ➾
- [ ] 🚀 Release ➾

### CODE REVIEW

**Review by hu3man — APPROVED** [review]

File: bin/build-all.sh [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
     echo '******* BUILDING ALL TAQUERIA PACKAGES *******'
     echo '**********************************************'
+
     echo ""
     echo '**********************************************'
-    echo "** Installing NPM dependencies"
-    npm install
-    npm run bootstrap
+    echo "** Checking Dependencies"
+    docker ps
```
Comment (danielelisi) [inline bin/build-all.sh:12]: What is this for?

Comment (AlirezaHaghshenas) [reply bin/build-all.sh:12]: Docker is used at the very last steps of `build-all`. So if the engine is not running, the user will be informed only after building everything and failing at the last step. With this change, they will be notified right after the run command `build-all`.

File: taqueria-flextesa-manager/package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	"main": "index.js",
 	"scripts": {
 		"build-package": "npx tsc -noEmit && npx esbuild --platform=node --bundle index.ts --outdir=.",
-		"build-docker": "TAQ_VERSION=`taq --version` BUILD=`taq --build`; docker build . -t \"ghcr.io/ecadlabs/taqueria-flextesa:${TAQ_VERSION}-${BUILD}\"",
+		"build-docker": "TAQ_VERSION=`../taq --version` BUILD=`../taq --build`; docker build . -t \"ghcr.io/ecadlabs/taqueria-flextesa:${TAQ_VERSION}-${BUILD}\"",
```
[the remark this answers is not in the document]
Comment (AlirezaHaghshenas) [reply taqueria-flextesa-manager/package.json:9]: We can. I'm not sure which one is better. Or even if there is a real difference for us.
What's your opinion?

[the remark this answers is not in the document]
Comment (jchenche) [reply taqueria-flextesa-manager/package.json:9]: If we can use git it'd be more desirable

**Review by jchenche — APPROVED** [review]
LGTM! Just left one comment but if it's not easy to do, it's no problem.

**Pull request merged**